Source/Core/UICommon/DiscordPresence.h

```diff
@@ -4,10 +4,24 @@
 
 #pragma once
 
+#include <functional>
+
 namespace Discord
 {
+using JoinFunction = std::function<void()>;
+
+enum class SecretType : char
```

Comment:
Why `char` as opposed to just not specifying the type?

Reply:
That was because the SecretType were just going to be letters or symbols like ``IPAddress`` was going to be ``= '1'`` and adding it to the secret would be done with a single static cast. However, I changed my mind (And I can't remember why) and removed the ``=`` stuff but somehow forgot to remove the char.